Resources/Textures/_NF/Structures/Wallmounts/posters.rsi/meta.json

```diff
@@ -2,13 +2,23 @@
     "version": 1,
     "license": "CC-BY-NC-SA-4.0",
     "copyright": "nf_poster1_legit by exincore and remixed from gentlebutter's Xeeenoooossss",
+    "copyright": "nf_poster1_contraband, nf_poster2_contraband, nf_poster3_contraband by erhardsteinhauer (discord/github)",
```

Comment:
![image](https://github.com/user-attachments/assets/b93cf470-e680-435e-9d69-3380f1fd6fb0)

Reply:
It works, but only because JSON does not define what happens if an object contains multiple keys. In order to not kill automated tools, please consider combining these into a single value, with, like, a comma or semicolon or something.

Reply:
> It works, but only because JSON does not define what happens if an object contains multiple keys. In order to not kill automated tools, please consider combining these into a single value, with, like, a comma or semicolon or something.

Keep in mind this exists on 1 more item in the repo right now.

Reply:
Sure, I'll change it since the string in this meta.json isn't long enough (yet) to be considered a problem.